### PR TITLE
[PIP] Implement Shaun, Father of Synths

### PR DESCRIPTION
**Pull request by alexander-novo**
Also a couple of minor changes to triggered abilities and token copy effect discussed on Discord.

No tests - the card is fairly simple. I did play with it several times and made sure every ability works.

#11324

### CODE REVIEW

**Comment by alexander-novo** [comment]
[[Shaun, Father of Synths]]

File: Mage/src/main/java/mage/abilities/effects/common/CreateTokenCopyTargetEffect.java [hunk 1]
```diff
@@ -154,6 +154,7 @@ protected CreateTokenCopyTargetEffect(final CreateTokenCopyTargetEffect effect)
         this.tokenPower = effect.tokenPower;
         this.tokenToughness = effect.tokenToughness;
         this.useLKI = effect.useLKI;
+        this.permanentModifier = effect.permanentModifier;
```
Comment (xenohedron) [inline Mage/src/main/java/mage/abilities/effects/common/CreateTokenCopyTargetEffect.java:157]: please update TODO comment for the field, to remove the part that is no longer relevant after this fix

File: Mage/src/main/java/mage/abilities/TriggeredAbility.java [hunk 2]
```diff
@@ -46,6 +46,8 @@ public interface TriggeredAbility extends Ability {
 
     boolean isOptional();
 
+    void setOptional();
```
Comment (xenohedron) [inline Mage/src/main/java/mage/abilities/TriggeredAbility.java:49]: How about returning a TriggeredAbility, for chaining purposes

**Review by xenohedron — APPROVED** [review]

**Comment by alexander-novo** [comment]
Fixed those up, thanks.

**Pull request merged**